recipes/data-management/cygnus/ha/readme.md

```diff
@@ -80,11 +85,22 @@ By now, the data sent by the script has been processed by cygnus and will be ava
 
 Having cygnus running as a service on a Docker Swarm cluster, scaling it can be achieved as with any other docker service. For more details, refer to the [Orion recipe](../../context-broker/ha/readme.md) to see how this can be done with Docker. Otherwise, refer to the [Docker service docs](https://docs.docker.com/engine/swarm/swarm-tutorial/scale-service/).
 
-## What if I wanted a different backend?
+## Customisations
+
+### What if I wanted a different backend?
 
 If you wanted to try a different backend for your cygnus deployment, there are 3 steps you need to follow.
 
 1. Configure your ```cygnus_agent.conf``` according to your needs. More info [in the docs](http://fiware-cygnus.readthedocs.io/en/latest/cygnus-ngsi/installation_and_administration_guide/configuration_examples/index.html).
 1. Update the ```docker-compose.yml```, specifically the environment variables configured for the cygnus service.
     For example, if you wanted to use MongoDB instead of MySQL, you'll need to use variables CYGNUS_MONGO_USER and CYGNUS_MONGO_PASS. For a complete list of required variables, refer to the [cygnus docs](http://fiware-cygnus.readthedocs.io/en/latest/cygnus-ngsi/installation_and_administration_guide/install_with_docker/index.html#section3.2).
 1. Update the ```docker-compose.yml```, removing the definition of the mysql service and introducing the one of your preference. Also, don't forget to update the ```depends_on:``` section of cygnus with the name of your new service.
+
+### Using a different channel
+
+If you take a look at the configuration file in `conf/cygnus_agent.conf`, you can choose between a Memory-based or a File-based channel. Feel free to comment/uncomment (i.e, leave/remove the `#` character) from the channel type configuration.
```

Comment:
refer to the section on ha for which one to chose and why

Reply:
there's already a reference to the section that explains channels considerations